main.cpp

```diff
@@ -243,238 +242,229 @@ struct Resturant
-    1) number of cars in inventory(int)
-    2) amount of cars sold in a month(int)
-    3) amount sales people on staff(int)
-    4) number of cars serviced each week(int)
-    5) amount of inventory bought this month(int)
-3 things it can do:
-    1) sell cars
-    2) hire salesmen
-    3) buy inventory
- */
+
+void Resturant::serveFood()
+{
+
+}
+
+float Resturant::collectPayment(float payment)
+{
+    return payment;
+}
+
+void Resturant::utensils()
+{
+
+}
+
+void Resturant::Server::serveCust()
+{
+
+}
+
+void Resturant::Server::cleanTable()
+{
+
+}
+
+float Resturant::Server::countTip(float tip)
+{
+    return tip;
+}
+
 struct CarDealership
 {
-    //1) number of cars in inventory(int)
     int carInventory = 200;
-    //2) amount of cars sold in a month(int)
     int carSold = 50;
-    //3) amount sales people on staff(int)
     int staff = 23;
-    //4) number of cars serviced each week(int)
     int carServiced = 40;
-    //5) amount of inventory bought this month(int)
     int invBought = 15;
-    //1) sell cars
-    float CarSold(float price);
-    //2) hire salesmen
+    float carSale(float price);
     void salesmen();
-    //3) buy inventory
-    int inventor(int invBought);
+    int invator(int invatory);
 };
-/*
-Thing 5) speakers
-5 properties:
-    1) brand (std::string)
-    2) model number (int)
-    3) size (int)
-    4) frequency range (float)
-    5) impedence (float)
-3 things it can do:
-    1) produce audio
-    2) receive signal
-    3) vibrate
- */
-struct Speakers
+
+float CarDealership::carSale(float price)
+{
+    return price;
+}
+
+void CarDealership::salesmen()
+{
+
+}
+
+int CarDealership::invator(int invatory)
 {
-    //1) brand (std::string)
+    return invatory;
+}
 
+struct Speakers
+{
+ 
     std::string brand = "Vagnavox";
-    //2) model number (int)
     int modelNum = 2340;
-    //3) size (int)
     int size = 12;
-    //4) frequency range (float)
     float freqRange = 145.4f;
-    //5) impedence (float)
     float impedence = 50.0f;
-    //1) produce audio
     void audio();
-    //2) receive signal
     void receiveSignal();
-    //3) vibrate
     void vibrate();
 };
-/*
-Thing 6) amp
-5 properties:
-    1) brand (std::string)
-    2) model number (int)
-    3) power (float)
-    4) impedence (float)
-    5) outputs (int)
-3 things it can do:
-    1) project audio signal
-    2) power speakers
-    3) protect system from failure
- */
+
+void Speakers::audio()
+{
+
+}
+
+void Speakers::receiveSignal()
+{
+    
+}
+
+void Speakers::vibrate()
+{
+    
+}
+
 struct Amp
 {
-    //1) brand (std::string)
     std::string brand = "Fender";
-    //2) model number (int)
     int modelNum = 1200;
-    //3) power (float)
     float power = 200.f;
-    //4) impedence (float)
     float impedence = 200.f;
-    //5) outputs (int)
     int outputs = 4;
-    //1) project audio signal
     void amplify();
-    //2) power speakers
     void speakers();
-    //3) protect system from failure
     void surgeProtect();
```

Comment:
I didn't catch these improper function names in part 1. 
What you have written here does not match your plain-english actions.
You need to include the action words in your function names
```
void projectAudioSignal();
void powerSpeakers();
void protectSystemFromFailure();
```
- [ ] revise